projects/train/train/cli/base.py

```diff
@@ -2,6 +2,12 @@
 
 
 class AmplfiBaseCLI(LightningCLI):
+    def __init__(self, *args, **kwargs):
+        # hack into init to hardcode
+        # parser_mode to omegaconf for all subclasses
+        kwargs["parser_kwargs"] = {"parser_mode": "omegaconf"}
```

Comment:
What feature was not being supported because of omegaconf?

Reply:
`omegaconf` adds parsing of environment variables (like `AMPLFI_OUTDIR`)